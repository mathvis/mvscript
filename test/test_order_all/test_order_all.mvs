1 + 2 * 3 b& 1 ^ 2
2 > 5 && true || false
