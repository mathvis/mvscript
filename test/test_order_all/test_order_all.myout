Expr (Type (Int 3))
Expr (Type (Bool False))
