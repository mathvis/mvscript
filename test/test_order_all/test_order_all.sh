#!/bin/bash
helix test_order_all.mvs test_order_all.out --vsplit
