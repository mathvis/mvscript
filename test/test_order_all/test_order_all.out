Expr (Operation (Or (Operation (And (Operation (GreaterThan (Operation (BitwiseXor (Operation (BitwiseAnd (Operation (Add (Type (Int 1)) (Operation (Multiply (Type (Int 2)) (Type (Int 3)))))) (Type (Int 1)))) (Type (Int 2)))) (Type (Int 5)))) (Type (Int 3)))) (Type (Int 2))))
